LykkeApi2/Modules/Api2Module.cs

```diff
@@ -1,115 +1,121 @@
-﻿using Autofac;

```

Comment:
madiff is too huge because of whitespaces
this should be fixed, in order to make history cleaner